deploy/kubernetes/setup.sh

```diff
@@ -182,9 +190,10 @@ kubectl -n $NAMESPACE create secret generic sumologic \
   --from-literal=endpoint-logs=$ENDPOINT_LOGS \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 
-echo "Applying deployment 'fluentd'..."
+echo "Applying deployment 'fluentd'... $release"
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
 sed 's/\$NAMESPACE'"/$NAMESPACE/g" | \
+sed 's/\0.0.0'"/$release/g" | \
```

Comment:
I think we'd eventually want to update the master yaml to also reflect the latest stable version, in which case we'd have to remember to change it here as well, not sure how I feel about that

Reply:
Agreed, I'm going to assume that we will bump the version in the master Yaml every so often.  Could we try to replace this based on a regex search, or at least looking for `image: sumologic/kubernetes-fluentd:` ?